src/pages/Calculator/Steps/MixRatios/index.jsx

```diff
@@ -289,6 +289,7 @@ const MixRatio = ({
             expanded={accordionState[seed.label]}
             onChange={() => handleExpandAccordion(seed.label)}
             summary={<Typography>{seed.label}</Typography>}
+            testId={`accordion-${seed.label}`}
```

Comment:
what's the difference between data-cy, data-test, and testId? is there a reason for using all 3?

Reply:
data-cy and data-test are both used for query I'm only using data-test in seedcalc.
I'm using testId as a prop here because this is a custom component, if directly set data-test will not be applied to the component.